docs/api/components.md

```diff
@@ -235,11 +235,17 @@ angular.component('myComponent', {
   controller: connect({
     foo: state`foo`,
     click: signal`clicked`
-  }, function MyController () {
+  }, 'myComponent', function MyController () {
     // Optionally add custom behaviour to controller
   })
 })
 ```
+Since angular doesn't expose the component name,
+you will need to provide one to `connect` for the
+component to be given a name in cerebral.
+
+Both component name and controller are optional arguments.
+You can provide either or both, and in any order.
```

Comment:
You mean "in any order"? I think we do because there is mostly no need to add controller. And not sure if we should force naming... cause we do not do that in Vue either.

Only change here would be to force naming the component, which could force argument order... but since both controller and name is optional they have to be flexible. Easier to say "any order" than "any order of last two arguments" :)